protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -633,7 +513,7 @@ struct AppStorage {
     Storage.TotalGerminating oddGerminating;
     Storage.TotalGerminating evenGerminating;
     // mapping from season => unclaimed germinating stalk and roots
-    mapping(uint32 => Storage.Sr) unclaimedGerminating;
+    mapping(uint32 => Storage.GerminatingSilo) unclaimedGerminating;
     Storage.WhitelistStatus[] whitelistStatuses;
     address sopWell;
```

Comment:
there's a extra `barnRaiseWell` natspec param that I assume used to be this variable. Can we remove it in natspec?

Reply:
fixed in 7ea86b8f732de1bd4bea2313c197882e15f888b6